app/controllers/application_controller.rb

```diff
@@ -1,2 +1,22 @@
 class ApplicationController < ActionController::Base
+  def render_resource(resource)
+    if resource.errors.empty?
```

Comment:
this can be done with a guard clause(`return x if y`)

Reply:
yes, it's the same, if I have more actions after the conditional with the return I break before, in this case it's the same.